fluid/sfont.cpp

```diff
@@ -80,12 +80,23 @@ bool SFont::read(const QString& s)
       if (!load())
             return false;
 
-      foreach(Instrument* i, instruments) {
+      synth->setLoadProgress(0);
+      int currentInstument = 0;
+      foreach(Instrument* i, instruments)
```

Comment:
coding style: please use C++11's `for (... : ...)` rather than Qt's `foreach(..., ...)`